my.new.project/XPathProject.txt

```diff
@@ -0,0 +1,7 @@
+1. $x("//input[@id='userName']")
+2. $x("//button")
+3. $x("//div[text()='Vega']")
```

Comment:
Narrow the context
//div[@class='ReactTable -striped -highlight']//div[text()='Vega']